### DISCUSSION TITLE
Chirpity Model Accuracy

### DISCUSSION THREAD

**Discussion by paulpeyret** [Mattk70/Chirpity-Electron#45] [Q&A] [answered]
Hi @Mattk70 ,
I just discovered Chirpity and all the potential it has for nocmig! First, I must tell that I am very impressed by the nice UI and the speed of the processing. 
I am currently trying to find the most accurate model for Mig Flight Calls detection in France.
Chirpity seems quite accurate from my first tests so it caught my curiosity! 
Is it a model you developed yourself? 
Was it trained on data collected only in UK or could it be used in France too?
Is there any Python script available to retrain this model on new data and improve accuracy even more?
Thanks a lot!

**Comment by Mattk70** [comment, marked as answer]
Hi Paul, thanks!

The native "Chirpity" model is one I developed myself, yes. It'll work just as well in France as the UK, so long as the species you're looking for are among the species it was trained on.  You can browse the species list if you go to the help section and click "What species are detected?" If you want all the birds species to appear in the "included" panel, select the "birds" list first, either from Settings or by clicking the icon on the filter panel until the bird icon appears. 

The Chirpity model cannot be retrainied, at least not by anyone, like BirdNET can be.  So if you wanted to train your own model, I would start with the [BirdNET ](https://github.com/kahst/BirdNET-Analyzer) one!

BirdNET's custom classifiers can be output as Raven  or tflite models.  It is possible, but not a simple process, to convert tflite to TFJS, which is what Chirpity uses. You can read up on that here:

https://discuss.tensorflow.org/t/how-to-convert-tflite-to-model-json-weights/17174/3

If you get that far, and want to import your model into Chirpity, come back and let me know!!

**Reply by paulpeyret** [reply]
Thanks for the advice and ressources!
I will first evaluate Chirpity model as is an then see if it suits my need.
